pkg/ndctl/ndctl.go

```diff
@@ -62,14 +67,19 @@ func (ctx *Context) GetBuses() []*Bus {
 
 //CreateNamespace create new namespace with given opts
 func (ctx *Context) CreateNamespace(opts CreateNamespaceOpts) (*Namespace, error) {
+	var err error
+	var ns *Namespace
 	for _, bus := range ctx.GetBuses() {
 		for _, r := range bus.ActiveRegions() {
-			if ns, err := r.CreateNamespace(opts); err == nil {
+			if ns, err = r.CreateNamespace(opts); err == nil {
+				glog.Infof("Namespace %s created in %s", ns.Name(), r.DeviceName())
 				return ns, nil
+			} else {
+				glog.Errorf("Namespace creation failure in %s: %s", r.DeviceName(), err.Error())
 			}
 		}
 	}
-	return nil, fmt.Errorf("Failed to create namespace")
+	return nil, errors.Wrap(err, "failed to create namespace")
```

Comment:
yes, as wrap does not nest errors, it can just add to error text, it's not easy to bring up all errors with last error here, so I decided to log earlier ones and return t he latest. In most cases the errors from different regions are expected same.

Reply:
Ok, just wanted to make sure.